test-requirements.txt

```diff
@@ -2,6 +2,7 @@
 httpretty>=0.7.1
 pytest
 pytest-cov
+pytest-mock
```

Comment:
As in add a new `build-depends` section to the `debian` section? Should we add other dependencies there too, like the reponses package we just added?

Reply:
We looked over packages/bddeb (which sources information via the command `./tools/read-dependencies --distro debian` during the local package build ./packages/bddeb.  What I neglected to recall is that bddeb grew the ability to also read [build dependencies from test-requirements.txt](https://github.com/canonical/cloud-init/blob/main/packages/bddeb#L84-L87). So, I don't think we need to add those deps in pkg-deps.json too since you already added it to test-requirements.txt.  You can validate that with python3-pytest-mock is listed in `./tools/read-dependencies --requirements-file test-requirements.txt --system-pkg-names`